calling an empty func

The following program `sample.go` triggers a panic:

```go
package main

func use(interface{}) {}

func main() {
        z := map[string]interface{}{"a": 5}
        use(z)
        println("bye")
}
```

Expected result:
```console
$ go run ./sample.go
bye
```

Got:
```console
$ yaegi ./sample.go
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/containous/yaegi/interp.runCfg.func1(0xc0001165a0, 0xc00011d500)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:113 +0x20c
panic(0x63d180, 0xc000016420)
        /home/marc/sdk/go1.14.1/src/runtime/panic.go:967 +0x166
github.com/containous/yaegi/interp.call.func5(0xc0001165a0, 0xc0001243f0)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:797 +0x104e
github.com/containous/yaegi/interp.runCfg(0xc00011d500, 0xc0001165a0)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:119 +0x6d
github.com/containous/yaegi/interp.genFunctionWrapper.func2.1(0x829a70, 0x0, 0x0, 0x829a70, 0xc000068f60, 0x5fffc0)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:542 +0x3a4
reflect.callReflect(0xc000069050, 0xc000145f30, 0xc000145f18)
        /home/marc/sdk/go1.14.1/src/reflect/value.go:549 +0x322
reflect.makeFuncStub(0x5fffc0, 0xc000069050, 0x13, 0x5fff01, 0x5fffc0, 0xc000069050, 0x0, 0x0, 0xc000001380, 0xc000145fd0, ...)
        /home/marc/sdk/go1.14.1/src/reflect/asm_amd64.s:20 +0x42
main.main()
        /home/marc/tmp/issue-312-1.go:32 +0x103
exit status 2
```

--- a/_test/fun12.go
+++ b/_test/fun12.go
@@ -1,0 +1,12 @@
+package main
+
+func use(interface{}) {}
+
+func main() {
+	z := map[string]interface{}{"a": 5}
+	use(z)
+	println("bye")
+}
+
+// Output:
+// bye

--- a/interp/run.go
+++ b/interp/run.go
@@ -762,39 +762,40 @@ func call(n *node) {
 		}
 
 		// Copy input parameters from caller
-		dest := nf.data[numRet:]
-		for i, v := range values {
-			switch {
-			case method && i == 0:
-				// compute receiver
-				var src reflect.Value
-				if v == nil {
-					src = def.recv.val
-					if len(def.recv.index) > 0 {
-						if src.Kind() == reflect.Ptr {
-							src = src.Elem().FieldByIndex(def.recv.index)
-						} else {
-							src = src.FieldByIndex(def.recv.index)
+		if dest := nf.data[numRet:]; len(dest) > 0 {
+			for i, v := range values {
+				switch {
+				case method && i == 0:
+					// compute receiver
+					var src reflect.Value
+					if v == nil {
+						src = def.recv.val
+						if len(def.recv.index) > 0 {
+							if src.Kind() == reflect.Ptr {
+								src = src.Elem().FieldByIndex(def.recv.index)
+							} else {
+								src = src.FieldByIndex(def.recv.index)
+							}
 						}
-					}
-				} else {
-					src = v(f)
-				}
-				// Accommodate to receiver type
-				d := dest[0]
-				if ks, kd := src.Kind(), d.Kind(); ks != kd {
-					if kd == reflect.Ptr {
-						d.Set(src.Addr())
 					} else {
-						d.Set(src.Elem())
+						src = v(f)
 					}
-				} else {
-					d.Set(src)
+					// Accommodate to receiver type
+					d := dest[0]
+					if ks, kd := src.Kind(), d.Kind(); ks != kd {
+						if kd == reflect.Ptr {
+							d.Set(src.Addr())
+						} else {
+							d.Set(src.Elem())
+						}
+					} else {
+						d.Set(src)
+					}
+				case variadic >= 0 && i >= variadic:
+					vararg.Set(reflect.Append(vararg, v(f)))
+				default:
+					dest[i].Set(v(f))
 				}
-			case variadic >= 0 && i >= variadic:
-				vararg.Set(reflect.Append(vararg, v(f)))
-			default:
-				dest[i].Set(v(f))
 			}
 		}
 

